src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Models/BankIdLoginApiInitializeResponse.cs

```diff
@@ -2,13 +2,20 @@
 {
     public class BankIdLoginApiInitializeResponse
     {
-        internal BankIdLoginApiInitializeResponse(bool isAutoLaunch, bool showLaunchButton, bool checkStatus, string orderRef, string redirectUri)
+        internal BankIdLoginApiInitializeResponse(
```

Comment:
As this is internal, we have full control, so I would skip the default`null` value.

Reply:
Sure! It was added as a way of skipping passing null in the builder methods.